pmm/pmm2-server-autobuild.groovy

```diff
@@ -0,0 +1,176 @@
+library changelog: false, identifier: 'lib@master', retriever: modernSCM([
+    $class: 'GitSCMSource',
+    remote: 'https://github.com/Percona-Lab/jenkins-pipelines.git'
+]) _
+
+pipeline {
+    environment {
+        DESTINATION = 'laboratory'
+    }
+    agent {
+        label 'large-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'PMM-2.0',
+            description: 'Tag/Branch for percona-images repository',
+            name: 'GIT_BRANCH')
+    }
+    options {
+        skipDefaultCheckout()
+        disableConcurrentBuilds()
+    }
+    triggers {
+        upstream upstreamProjects: 'pmm2-submodules-rewind', threshold: hudson.model.Result.SUCCESS
+    }
+    stages {
+        stage('Prepare') {
+            steps {
+                installDocker()
+
+                git poll: true, branch: GIT_BRANCH, url: 'http://github.com/Percona-Lab/pmm-submodules'
+                sh '''
+                    git reset --hard
+                    sudo git clean -xdf
+                    git submodule update --init --jobs 10
```

Comment:
there was an issue due to lfs enabled in git. Is it works correctly with such commands now?

Reply:
@EvgeniyPatlan Can you update git and install git lfs globally? What the other change did (wget'ing lfs, etc.) is an ad-hoc solution, and we would like to remove it.

Reply:
yes it seems to works, I tested with a Job